doc/index.rst

```diff
@@ -8,6 +8,7 @@
 
    introduction
    getting_going
+   examples
```

Comment:
Please remove commented out examples section further down in this file.

Reply:
Also, should we add a copyright notice and licence to this too.

Reply:
Well spotted and done.